packages/errors/package.json

```diff
@@ -10,11 +10,11 @@
   "engines": {
     "node": ">=10.16.3"
   },
-  "main": "./dist/index.js",
-  "types": "./dist/index.d.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "files": [
-    "./dist/**/*.js",
-    "./dist/**/*.d.ts"
+    "dist/**/*.js",
```

Comment:
Relative paths don't work, so this was updated so the prepare script would work.